.gflows/workflow-configuration/build-publish/settings.yml

```diff
@@ -68,6 +69,14 @@ service: #docker-build-and-job
   generate_docker_meta: true
   output_docker_digest: true
```

Comment:
This should be added to the new section so that digest related issues on comments below get resolved.

Reply:
Done